app/gui/src/controller/searcher/component/hardcoded.rs

```diff
@@ -106,10 +91,8 @@ impl Snippet {
     /// documentation parser cannot be created or the argument fails to parse as valid
     /// documentation.
     fn with_documentation(mut self, documentation: &str) -> Self {
-        let doc_parser = parser_scala::DocParser::new().unwrap();
-        let doc_string = documentation.to_string();
-        let documentation_html = doc_parser.generate_html_doc_pure(doc_string);
-        self.documentation_html = Some(documentation_html.unwrap());
+        // TODO: we need the Scala parser in searcher tests only for HTML documentation.
```

Comment:
This doesn't look like the correct change, but removing the doc parser usage allows to turn all searcher tests into native ones. This is a huge advantage while developing, though I probably should revert this afterwards and make searcher tests `wasm_test` again.

Reply:
At the same time, the documentation for  snippets is extremely primitive and doc parser doesn't really bring much.

Reply:
We can stay with this change. Before the searcher, tests were usually wasm_bindgen anyway, because we used the Scala parser. But I guess it's no longer the case?

Reply:
Yep, now the tests use Rust parser in general.

Reply:
I decided to remove the scala parser from here, it does not bring us much.